modules/snooker/src/main/ts/ephox/snooker/model/Fitment.ts

```diff
@@ -65,12 +65,15 @@ const generateElements = (amount: number, row: Structs.RowCells, generators: Sim
   return Arr.range(amount, (idx) => Structs.elementnew(generator(), true, isLocked(idx)));
 };
 
-const rowFill = (grid: Structs.RowCells[], amount: number, generators: SimpleGenerators, lockedColumns: Record<string, boolean>): Structs.RowCells[] =>
-  grid.concat(Arr.range(amount, () => {
-    const row = grid[grid.length - 1];
+const rowFill = (grid: Structs.RowCells[], amount: number, generators: SimpleGenerators, lockedColumns: Record<string, boolean>): Structs.RowCells[] => {
+  const exampleRow = grid[grid.length - 1];
```

Comment:
If this is fixing a bug that's visible at the editor level, should it get a TinyMCE changelog entry?

(to me who doesn't know snooker, it _looks like_ this is where the core of the fix is for that row information being lost bug is, if not then pretend I made the comment in the right place)

Reply:
You're correct. I'll add an editor changelog, thanks!

Reply:
Fixed